tests/test_cli_weekly_report.py

```diff
@@ -20,29 +42,120 @@ def test_weekly_report_image_generation():
 
         target_dir = Path(".") / "downloads"
         shutil.copytree(fixtures_dir, target_dir)
+        yield runner
+
+
+def test_weekly_report_image_generation(setup_cli_isolated_env):
+    runner = setup_cli_isolated_env
+
+    result = runner.invoke(
+        cli,
+        [
+            *BASE_ARGS,
+            "--output-type",
+            "images-only",
+        ],
+    )
+
+    assert result.exit_code == 0, result.output
+    assert Path(
+        f"{OUTPUT_DIR}/my-awesome-team-team-productivity-dashboard.png"
+    ).exists()
```

Comment:
Because we're running in an "isolated file system", we are sure that this path does _not_ exist when the test starts running...right?  (Otherwise, this is a pretty weak assertion....)

Reply:
Yes :-)